types/resize-observer-browser/index.d.ts

```diff
@@ -1,25 +1,23 @@
 // Type definitions for non-npm package resize-observer-browser 0.1
 // Project: https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserver, https://developers.google.com/web/updates/2016/10/resizeobserver, https://wicg.github.io/ResizeObserver/
-// Definitions by: Chives <https://github.com/chivesrs>
+// Definitions by: Chives <https://github.com/chivesrs>, William Furr <https://github.com/wffurr>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.7
 
-declare global {
-    interface Window {
-        ResizeObserver: typeof ResizeObserver;
-    }
+interface Window {
+    ResizeObserver: typeof ResizeObserver;
 }
 
-export class ResizeObserver {
+declare class ResizeObserver {
```

Comment:
Oh, `.d.ts` files implicitly add declare before every single item. Do you need this here? Or does everything work without the declare keyword?

Reply:
If I remove "declare" from the class declaration, I get the following error:

index.d.ts:11:1 - error TS1046: A 'declare' modifier is required for a top level declaration in a .d.ts file.

Reply:
All of the handbook examples include declare on namespace, class, and var.

Reply:
Huh, ok. I thought `.d.ts` implied declare automatically. Guess I was wrong.